### PR TITLE
[ESIMD] Add a test case checking SYCL_RT_WARNING_LEVEL usage

### PR DESCRIPTION
**Pull request by v-klochkov**
The vc-codegen is expected to get -disable-finalizer-msg switch
when the SYCL_RT_WARNING is not set to 1 or a greater number.

Signed-off-by: Vyacheslav N Klochkov <vyacheslav.n.klochkov@intel.com>

### CODE REVIEW

File: SYCL/ESIMD/vadd_1d.cpp [hunk 1]
```diff
@@ -8,7 +8,8 @@
 // REQUIRES: gpu
 // UNSUPPORTED: cuda || hip
 // RUN: %clangxx -fsycl %s -o %t.out
-// RUN: env SYCL_PI_TRACE=-1 %GPU_RUN_PLACEHOLDER %t.out 2>&1 %GPU_CHECK_PLACEHOLDER
+// RUN: env SYCL_PI_TRACE=-1 %GPU_RUN_PLACEHOLDER %t.out 2>&1 %GPU_CHECK_PLACEHOLDER --check-prefix CHECK-W0
+// RUN: env SYCL_PI_TRACE=-1 SYCL_RT_WARNING_LEVEL=1 %GPU_RUN_PLACEHOLDER %t.out 2>&1 %GPU_CHECK_PLACEHOLDER --check-prefix CHECK-W1
```
Comment (kbobrovs) [inline SYCL/ESIMD/vadd_1d.cpp]: Does it make sense to add a test that would check that stderr and stdout is empty even if there are spills?

**Review by kbobrovs — DISMISSED** [review]

**Review by kbobrovs — COMMENTED** [review]
Sorry, hit wrong "approve" button

File: SYCL/ESIMD/vadd_1d.cpp (hunk 1 above)
Comment (v-klochkov) [reply SYCL/ESIMD/vadd_1d.cpp]: Surprisingly, I found it difficult to have a test that would reliably give warnings with SYCL_RT_WARNING_LEVEL=1.
After recent updates (I am not sure if that is LLVM/SYCL or GPU RT), majority of tests that printed those warnings stopped doing so.

Regarding 'does it make sense to add a test...' part... 
The current patch checks that SYCL passes the option to GPU RT and that seems sufficient. 
Adding a test that checks stdout/err is empty with -disable-finalizer-msg means to check that GPU RT honors their own flag, which seems a bit excessive to me.
I can though add a test if you think it is needed.

Comment (kbobrovs) [reply SYCL/ESIMD/vadd_1d.cpp]: mm, maybe you are right.

**Review by kbobrovs — DISMISSED** [review]
